src/streaming/utils/ErrorHandler.js

```diff
@@ -55,12 +55,12 @@ function ErrorHandler() {
 
     // "mediasource"|"mediakeys"
     function capabilityError(err) {
-        eventBus.trigger(Events.ERROR, {error: 'capability', event: err});
+        eventBus.trigger(Events.ERROR, {error: 'capabilityError', event: err});
```

Comment:
Changing any of these would be a breaking change to applications listening for particular types of error. Not a reason not to change them, but worth bearing in mind ...

Reply:
I totally agree with you...